.buildkite/metricbeat/scripts/win_unit_tests.ps1

```diff
@@ -0,0 +1,70 @@
+$ErrorActionPreference = "Stop" # set -e
+$WorkFolder = "metricbeat"
+# Forcing to checkout again all the files with a correct autocrlf.
+# Doing this here because we cannot set git clone options before.
+function fixCRLF {
+    Write-Host "-- Fixing CRLF in git checkout --"
+    git config core.autocrlf false
+    git rm --quiet --cached -r .
+    git reset --quiet --hard
+}
+function withChoco {
+  Write-Host "-- Configure Choco --"
```

Comment:
**nit**: identation is not 4 spaces but 2, no clue what's the standard one though

Reply:
thanks, fixed!